README.md

```diff
@@ -59,7 +58,241 @@ To support these changes, there have been a number of changes to the SubnetEVM b
 
 ## Create an EVM Subnet on a Local Network
 
-See [this tutorial](https://docs.avax.network/subnets/create-a-local-subnet).
+### Clone Subnet-evm
+
+First install Go 1.17.9 or later, however as the time of writing, please don't use Go v1.18.x versions. Follow the instructions [here](https://golang.org/doc/install). You can verify by runing `go version`.
+
+Set `$GOPATH` environment variable properly for Go to look for Go Workspaces. We recommend `~/workspace` on a Mac. You can verify by running `echo $GOPATH`.
```

Comment:
I think we should really just point towards the golang instructions for setting up the GOPATH. I don't personally use `~/workspace`.

I'd recommend referring to the official golang docs, there's a section on setting up PATH and GOPATH correctly here: https://go.dev/doc/gopath_code.

Reply:
Additionally, since we have goreleaser now, only developers will need to actually go through this and probably won't have as much trouble setting up their go environment correctly as previously.